src/Compilers/CSharp/Test/Emit2/Semantics/ExtensionTypeTests.cs

```diff
@@ -15173,76 +15349,76 @@ public static void M(int i)
     }
 
     [ConditionalFact(typeof(CoreClrOnly))]
-    public void ExtensionInvocation_InstanceReceiver_Simple()
+    public void ExtensionInvocation_TypeReceiver_Simple_ExpressionTree()
     {
-        // We look at methods in extension types if the method group on type has no applicable candidates
+        // PROTOTYPE decide whether to allow expression tree scenarios
         var source = """
-new C().M(42);
+using System.Linq.Expressions;
+Expression<System.Action> x = () => C.M(42);
 
 class C
 {
-    public void M() => throw null;
+    public static void M() => throw null;
 }
 
 implicit extension E for C
 {
-    public void M(int i)
+    public static void M(int i)
     {
         System.Console.Write("E.M");
     }
 }
 """;
         var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
 
-        // PROTOTYPE Execute when adding support for emitting non-static members
-        //CompileAndVerify(comp, expectedOutput: "E.M", verify: Verification.FailsPEVerify);
+        CompileAndVerify(comp, expectedOutput: "", verify: Verification.FailsPEVerify).VerifyDiagnostics();
```

Comment:
>"" [](http://example.com/codeflow?start=47&length=2)

I think we should assert the shape of the expression tree. A PTOTOTYPE comment is sufficient for now.